README.md

```diff
@@ -179,7 +180,20 @@ type Msg = FilterStateChanged { event : { type_ : String }, states : Dict String
 AgGrid.grid { gridConfig | filterState = model.filterState } [ onFilterStateChanged FilterStateChanged ] [] []
 ```
 
-# Master-Detail
+## Value formatting
+
+The package allows defining `valueGetter`, `valueFormatter`, `valueParser`, `valueSetter`, and `filterValueFormatter` as expressions for formatting cell values.
+These make it possible to change the type of the cell value the table works with (e.g. strings parsed as numbers to allow aggregations - valueGetter), to format values displayed to the user in the table/filter (e.g. with thousand separators and currency symbol - valueFormatter/filterValueFormatter), and to parse user input before applying the value (e.g. parsing a formatted currency value back into a floating-point number string - valueParser).
+
+**For a detailed explanation of what the expressions can do, refer to the AgGrid documentation.**
+
+The package provides preconfigured value formatting for currencies, decimals, and percentages. Those are available as a `Renderer` type and already configure a certain `valueFormatter`, `valueGetter`, `filterValueFormatter`, and `cellEditor`. If required, the formatting can be overwriten by setting a `valueFormatter`/`filterValueFormatter` yourself on the `ColumnDef`. Also, these predefined expressions can be applied individually to other `Renderer` types if necessary. These are available in the `AgGrid.ValueFormat` module.
+
+- **CurrencyRenderer** to format floating-point strings into localized currency strings (e.g. '1000.15' --> '1.000,15 €')
+- **DecimalRenderer** to format floats into localized strings (e.g. 1000.8 --> '1.000,8')
```

Comment:
I thought about it again. It is probably saver to use strings after all because we can't loose precision. And converting a float to a string is easy anyway. So I think all the renderers should expect strings.

Reply:
Is there even a risk of losing precision? I mean, we are just displaying the number. We don't do any calculations on the numbers.

Isn't it just a matter of convenience?

Doing an extra `String.fromFloat` on the columDef in the view and something like `Decode.string |> Decode.map String.toFloat` in the change decoder for each Decimal/Percent column actually feels a bit annoying when there's not really a reason for it. For currencies, I kept this as strings because that's the usual data type used (easy to convert, but also more representable as it can represent the full cents `"19.10" vs. 19.1`).